Main.java

```diff
@@ -9,7 +11,28 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user imput
+    Scanner input = new Scanner(System.in);
+    
+    // get the user's first numbers
+    System.out.println("Please enter an integer:");
+    int a = input.nextInt();
+
+    // get the user's second numbers
+    System.out.println("Please enter another integer:");
+    int b = input.nextInt();
```

Comment:
Good work. Like the previous question, a and b would be poor choices for variable names. Remember, variables should always be named using camelCase and describe what they are storing or being used for.